Docs/Workflows.md

```diff
@@ -39,6 +39,16 @@ jobs:
       timeout-minutes: 10
 ```
 
+## Check Spelling workflow
```

Comment:
Similar to the other workflows, add an example of how to use this from another repo.